src/mlpack/methods/ann/loss_functions/mean_bias_error_impl.hpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file mean_bias_error_impl.hpp
+ * @author Saksham Rastogi
+ *
+ * Implementation of the mean bias error performance function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LOSS_FUNCTION_MEAN_BIAS_ERROR_IMPL_HPP
+#define MLPACK_METHODS_ANN_LOSS_FUNCTION_MEAN_BIAS_ERROR_IMPL_HPP
+
+
+// In case it hasn't yet been included.
+#include "mean_bias_error.hpp"
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+template<typename InputDataType, typename OutputDataType>
+MeanBiasError<InputDataType, OutputDataType>::MeanBiasError()
+{
+  // Nothing to do here.
+}
+
+template<typename InputDataType, typename OutputDataType>
+template<typename InputType, typename TargetType>
+double MeanBiasError<InputDataType, OutputDataType>::Forward(
+    const InputType&& input, const TargetType&& target)
+{
+  return arma::accu(target-input) / target.n_cols;
+}
+
+template<typename InputDataType, typename OutputDataType>
+template<typename InputType, typename TargetType, typename OutputType>
+void MeanBiasError<InputDataType, OutputDataType>::Backward(
+    const InputType&& input,
+    const TargetType&& target,
+    OutputType&& output)
+{
+  output = -1;
```

Comment:
Hi @codeboy5, This is will make output = -1 , a single value however we would want it to have same size as input. You can try output.set_size(arma::size(input)) and use fill to fill it with -1, or any other way you feel comfortable with.

Reply:
how do use we fill ?

Reply:
```
output.set_size(arma::size(input));
output.fill(-1.0);
```